documentation.html

```diff
@@ -62,7 +62,7 @@
                                 <li><a href="documentation/storm-solr.html">Solr</a></li>
                                 <li><a href="documentation/storm-eventhubs.html">Azure EventHubs</a></li>
                                 <li><a href="documentation/flux.html">Flux</a> (declarative wiring/configuration of Topologies)</li>
-                            </ul>
+                                <li><a href="documentation/storm-sql.html">SQL</a> (writing topology in SQL)</li>                            </ul>
```

Comment:
minor, put </ul> in new line, and to be consistent, "writing topology" should be "writing topologies"?